libzeth/circuits/joinsplit.tcc

```diff
@@ -36,33 +36,33 @@ private:
     // field elements to which are added the public value of size 64 bits
     const size_t length_bit_residual =
         2 * (ZETH_V_SIZE * 8) +
-        digest_len_minus_field_cap * (1 + 2 * NumInputs + NumOutputs);
+        digest_len_minus_field_cap * (1 + 2 * NumInputs);
     // Number of field elements needed to pack this number of bits
     const size_t nb_field_residual =
         libff::div_ceil(length_bit_residual, FieldT::capacity());
 
-    // Multipacking gadgets for the inputs (nullifierS, commitmentS, hsig,
-    // message authentication tags (h_is) and the residual bits (comprising the
+    // Multipacking gadgets for the inputs (nullifierS, hsig, message
+    // authentication tags (h_is) and the residual bits (comprising the
     // previous variables' bits not containable in a single field element as
-    // well as the public values) (the root is a field element) because we pack
-    // the nullifiers (Inputs of JS = NumInputs), the commitments (Output of JS
-    // = NumOutputs) AND the signature hash h_sig (+1) AND the message
-    // authentication tags h_iS (+ NumInputs) AND the residual field elements
+    // well as the public values) (the root and cms are a field element)
```

Comment:
`(the root and cms are a field element)` -> (the root and cms are field elements)

Reply:
Updated.